clustermesh/clustermesh.go

```diff
@@ -1294,6 +1305,15 @@ func (k *K8sClusterMesh) Status(ctx context.Context) (*Status, error) {
 
 	s.Connectivity, err = k.statusConnectivity(ctx)
 
+	if k.params.Output == "json" {
+		jsonStatus, err := json.MarshalIndent(s, "", " ")
+		if err != nil {
+			return nil, fmt.Errorf("failed to marshal status to JSON")
+		}
+		fmt.Println(string(jsonStatus))
+		return nil, err
```

Comment:
I think this should be

```suggestion
		return s, nil
```

for consistency.